content/app/development/ux/fields/grouping/alternatives/_index.en.md

```diff
@@ -54,6 +54,11 @@ and as long as the filter does not match, the element will not be displayed.
 
 Determines whether the "Add new" button is displayed below the table. It is useful to hide this if you only wish to display data.
 
+### alwaysShowAddButton
+
+Determines whether the "Add new" button is displayed below the table when a row is open. If the property is set ``"alwaysShowAddButton": true`` the button would be visible as long the number of rows has not reached its upper limit, [maxCount.](http://localhost:1313/nb/app/development/ux/fields/grouping/setup/)
+For example, it could be nice to display the button in cases where [openByDefault](http://localhost:1313/nb/app/development/ux/fields/grouping/alternatives/#openbydefault) er satt til noe annet enn "false".
```

Comment:
1. You could probably add that this setting never works if `addButton` has been set to `false`, to reduce confusion between these two similar-sounding settings.
2. `er satt til noe annet enn "false".` wasn't properly translated to english
3. "It could be nice" is usually not the reason app developers use a certain configuration, and "something other than false" is true, isn't it? 🤔

So my suggestion for rewriting the second line here:
> A use-case for this setting is to allow users to add more rows without clicking to save and close existing rows first, which is especially useful in combination with `openByDefault` (where it might not be obvious to the user that more rows can be added).